### DISCUSSION TITLE
What does the Done mean in a question?

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#122363] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Two weeks ago, I asked a question in the GitHub Copilot forum. I got some feedback, but the question still isn't answered, and no one is responding to it anymore. I took a look at the question and saw there's a button with the text "Done" in it, and a checkmark next to Done. Does that mean that I've marked the question is one that no longer needs any feedback, even though I still don't have an answer? If that's not what Done means, then what does it mean?

**Comment by tstran155** [comment, marked as answer]
Hi rfalanga,

If you see a checkmark next to the 'Done' button, it is likely that your question has been resolved or completed. If your question is still unanswered or you need further discussion on it, look for an option to undo this action, for example by clicking the "Done" button again or removing the checkmark.
